### ISSUE TITLE
NoMethodError (undefined method `saved_change_to_attribute?' with Mongoid

### ISSUE THREAD

**Issue by honeypc** [lynndylanhurley/devise_token_auth#1335]
*Version
 - gem 'devise_token_auth', github: 'lynndylanhurley/devise_token_auth'
 - Rails 6
 - Devise 4.7
 Thanks for help

**Comment by VardanAggarwal** [comment]
I am facing the same issue. Did you find any resolution?

**Comment by simukappu** [comment]
I am facing the same issue with Rails 6 and Mongoid ORM.
It is not good solution, but it works by defining own *saved_change_to_attribute?* method in *User* model.
```ruby
require 'mongoid'
require 'mongoid-locker'
class User
  include Mongoid::Document
  include Mongoid::Timestamps
  include Mongoid::Locker

  devise :database_authenticatable, :confirmable
  include DeviseTokenAuth::Concerns::User
  # Devise
  ## Database authenticatable
  field :email,                type: String, default: ""
  field :encrypted_password,   type: String, default: ""
  ## Confirmable
  field :confirmation_token,   type: String
  field :confirmed_at,         type: Time
  field :confirmation_sent_at, type: Time
  ## Required
  field :provider,             type: String, default: "email"
  field :uid,                  type: String, default: ""
  ## Tokens
  field :tokens,               type: Hash,   default: {}

  def saved_change_to_attribute?(attr_name, **options)
    true
  end
end
```

**Comment by jattoabdul** [comment]
Any fix for this yet? Adding the "saved_cahnge_to_atribute?" method in models is looking too hacky.

**Comment by MaicolBen** [comment]
This was fixed in #348 and it's included in v1.1.4
